core/src/main/java/org/apache/shiro/env/BasicIniEnvironment.java

```diff
@@ -0,0 +1,22 @@
+package org.apache.shiro.env;
```

Comment:
@ddold can you add license header please?

Reply:
@fpapon This PR can be closed IMO. BasicIniEnvironment will be added as part of https://github.com/apache/shiro/pull/157, and the Quickstart no longer uses the IniSecurityManagerFactory

Reply:
@coheigea you're right, thanks!